test-cases/upgrades/multi-5-dc-rolling-upgrade.yaml

```diff
@@ -0,0 +1,31 @@
+test_duration: 3600
```

Comment:
2.5 days? Need to reduce it to some sane value.

Reply:
set a bit less. staging jobs take about 1d for upgrad